src/main/java/net/imglib2/util/Localizables.java

```diff
@@ -47,12 +47,14 @@
 
 public class Localizables
 {
-
 	public static long[] asLongArray( final Localizable localizable )
 	{
-		final long[] result = new long[ localizable.numDimensions() ];
-		localizable.localize( result );
-		return result;
+		return localizable.locationToLongArray();
+	}
+
+	public static double[] asDoubleArray( final RealLocalizable realLocalizable )
```

Comment:
may be deprecate because now we have the more convenient defaults

Reply:
Because I added this one as part of the PR, I'll remove instead of deprecating.